### PR TITLE
Machine Tokens - Update Links to New/Legacy Dashboard

### LINKED ISSUE #7721: Machine Tokens - Update Links
Re: [Creating and Revoking Machine Tokens](https://pantheon.io/docs/machine-tokens)

Priority:

## Issue Description:
Update machine tokens links on appropriate page(s); specify which link for which dashboard (legacy vs new).

### PR DESCRIPTION
**Pull request by michellecolon-pantheon**
Closes #7721 

## Summary

**[Creating and Revoking Machine Tokens](https://pantheon.io/docs/machine-tokens)** - Update machine tokens links on appropriate page(s); specify which link for which dashboard (legacy vs new).

## Effect

The following changes are already committed:

*
*

## Remaining Work and Prerequisites

The following changes still need to be completed:

- [ ] List any outstanding work here

### Dependencies and Timing

- [ ] Other prerequisites that must be completed before merging this PR

**Release**:
- [ ] When ready
- [ ] After date: $DATE

--------------------------------------------------

## Post Launch

**Do not remove** - To be completed by the docs team upon merge:

- [ ] Redirect `/docs/old-path/` => `/docs/new-path/` (if applicable)
- [ ] Include/exclude pages ^ respectively within docs search service provider (if applicable)
- [ ] For Heroes - add a props post to the [discussion board](https://discuss.pantheon.io/c/pantheon-platform/documentation/17).
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | [Live](https://pantheon.io/docs) | [pr-7722](https://pr-7722--pantheon-docs.my.pantheonfrontend.website/docs) | Difference |
| --- | --- | --- | --- |
| Performance Score | `49` | `53` | `4` |
| Best Practice Score | `86` | `86` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `98` | `98` | `0` |

Lighthouse Audit Reports:

- [`pr-7722` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/42105/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/42105/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)


[Preview Environment](https://pr-7722--pantheon-docs.my.pantheonfrontend.website/docs)

**Comment by michellecolon-pantheon** [comment]
[Preview](https://pr-7722--pantheon-docs.my.pantheonfrontend.website/docs/machine-tokens)

File: source/partials/machine-tokens-new-dash.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<Alert title="Machine Tokens in the New Dashboard" type="info">
+
+If you are using the [New Dashboard](https://pantheon.io/docs/guides/new-dashboard), you can visit your Personal Settings page to [create Machine Tokens](https://dashboard.pantheon.io/personal-settings/machine-tokens).
```
Comment (EdwardAngert) [inline source/partials/machine-tokens-new-dash.md:3]:
```suggestion
If you are using the [New Dashboard](/guides/new-dashboard), you can visit your Personal Settings page to [create Machine Tokens](https://dashboard.pantheon.io/personal-settings/machine-tokens).
```

**Review by EdwardAngert — CHANGES_REQUESTED** [review]

**Comment by wordsmither** [comment]
lgtm

**Pull request merged**